boxsdk/object/__init__.py

```diff
@@ -2,19 +2,6 @@
 
 from __future__ import unicode_literals
 
-import six
 
+__all__ = list(map(str, ['collaboration', 'events', 'file', 'folder', 'group', 'group_membership', 'search', 'user']))
```

Comment:
You can fix this with either

    from six.moves import map  # pylint:disable=redefined-builtin

or

    __all__ = [str(s) for s in ['collaboration', 'events', 'file', 'folder', 'group', 'group_membership', 'search', 'user']]